### ISSUE TITLE
 AIChatbot: Stream states and changes

### ISSUE THREAD

**Issue by odyss3y** [SilenusTA/StreamRoller#44]
Twitch examples would be:
Current Category: the bot knows what game/activity is currently being played
Category change: the bot is prompted that the previous game isnt being played anymore and there is a new category
join/parts: never ever use

**Comment by SilenusTA** [comment]
Good idea, while adding in the game to the personality isn't a problem, the act of getting it I still have to look into. I'm not sure if that comes through on the current API as I haven't tested it but if it does it should be a quick thing to add. 
When you say join/parts do you mean the text that comes up in SR chat window when the IRC joins/parts a channel?

**Comment by odyss3y** [comment]
No just stating that I don't think there should be triggers for join/part on twitchchat.

**Comment by SilenusTA** [comment]
Can now be done from the twitch stream game changed triggers

**Comment by SilenusTA** [comment]
Closing as should have been closed on the previous comment.
